community/how-to-release.md

```diff
@@ -520,11 +520,11 @@ title:
 [RESULT][VOTE] Release Apache Linkis ${release_version} ${rc_version}
 
 content:
-Hello Apache Linkis PMC and Community,
+Hello Apache Linkis PMC members and Community,
 
     The vote closes now as 72hr have passed. The vote PASSES with
```

Comment:
`Hello Apache Linkis PMC and Community` here I think is ok

Reply:
OK